data/net/minecraft/client/renderer/block/model/BlockModelDefinition.mapping

```diff
@@ -1,7 +1,18 @@
 CLASS net/minecraft/client/renderer/block/model/BlockModelDefinition
+	METHOD <init> (Ljava/util/List;)V
+		ARG 1 modelDefinitions
+	METHOD <init> (Ljava/util/Map;Lnet/minecraft/client/renderer/block/model/multipart/MultiPart;)V
+		ARG 1 variants
+		ARG 2 multiPart
+	METHOD equals (Ljava/lang/Object;)Z
+		ARG 1 obj
```

Comment:
This isn't required but generally recommend: `other` over `object` for `equals` methods. 
```suggestion
		ARG 1 other
```

Reply:
`other` and `object` are fine. `obj` should be changed per [Standards 4.](https://parchmentmc.org/docs/mappings.html)